src/docfx/restore/RestoreFile.cs

```diff
@@ -12,15 +15,24 @@ internal static class RestoreFile
     {
         public static async Task Restore(string url, Config config, bool @implict)
         {
-            if (implict && RestoreMap.TryGetFileRestorePath(url, out _))
+            if (RestoreMap.TryGetFileRestorePath(url, out var existingPath) && implict)
```

Comment:
>implict [](start = 79, length = 7)

why change the order?

Reply:
Before, we needn't call `RestoreMap.TryGetFileRestorePath` when `implicit == false`, but it is needed now.

Reply:
we either don't need do that, just split it into two `if`

---
In reply to: [239327888](https://github.com/dotnet/docfx/pull/3726#discussion_r239327888) [](ancestors = 239327888)